### PR TITLE
networkd: add quickack option to  route

### PR DESCRIPTION
**Pull request by ssahani**
This patch adds quickack option to enable/disable TCP quick ack
mode for per-route.

### CODE REVIEW

File: man/systemd.network.xml [hunk 1]
```diff
@@ -1095,6 +1095,13 @@
             </para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>QuickAck=</varname></term>
+          <listitem>
+            <para>Takes a boolean argument. When true enables TCP quick ack mode for per-route. Defaults to unset.
```
Comment (yuwata) [inline man/systemd.network.xml]: for per-route. -> for the route.

**Comment by ssahani** [comment]
updated

**Comment by ssahani** [comment]
updated

File: src/network/networkd-route.c [hunk 2]
```diff
@@ -648,6 +648,12 @@ int route_configure(
                         return log_error_errno(r, "Could not append RTAX_INITRWND attribute: %m");
         }
 
+        if (route->quickack != -1) {
```
Comment (yuwata) [inline src/network/networkd-route.c:652]: What is the default value of `quickack`? Where is it initialized to -1?

Comment (ssahani) [reply src/network/networkd-route.c:652]: right fixed.

**Pull request merged**